.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -87,46 +144,42 @@ body:
     validations:
       required: true
   - type: markdown
+    id: general-information-logs
     attributes:
       value: |
-        When providing logs, please keep the following things in mind.
+        When providing logs, please keep the following things in mind:
         1. **DO NOT** use external paste services.
         2. Please provide complete logs.
           - For server logs, include everything you think is important plus *10 lines before and after*
```

Comment:
I think it would be better to change this to be based on the time of the event rather than "looks important"